tests/model/test_model.py

```diff
@@ -1430,6 +1430,38 @@ def test_fetch_raw_message_content(
         assert self.display_error_if_present.called == display_error_called
         assert return_value == expected_raw_content
 
+    @pytest.mark.parametrize(
+        "response, expected_read_receipts, display_error_called",
+        [
+            (
+                {"msg": "", "result": "success", "user_ids": [3, 7, 9]},
+                "Verona, alice, Bob",
+                False,
+            ),
+            (
+                {"msg": "Invalid message(s)", "result": "error"},
+                None,
+                True,
+            ),
+        ],
+    )
+    def test_fetch_read_receipts(
+        self,
+        mocker,
+        model,
+        expected_read_receipts,
+        response,
+        display_error_called,
+        message_id=1,
+    ):
+        self.client.get_read_receipts = mocker.Mock(return_value=response)
```

Comment:
In the method you've written, you're not using `Client.get_read_receipts`?
Instead you're using `.call_endpoint`?

Reply:
I couldn't find a function that did that. Thought there wasn't one so directly called the endpoint. 
Also, I wrote that so that I could add an attribute to the client with `get_read_receipts` as name and then add the parameterised response as the value hit returns hoping it would help me solve the test cases errors

Reply:
To be clear, it's absolutely fine to use the `.call_endpoint` function if there isn't a named version available (you might try adding one via a PR in that repo) - and using it more generally will also work.

The problem here is that in the test, you're giving the `client` object a `.get_read_receipts` attribute, and then calling the function you've defined. That assigns a value based on `.call_endpoint`, which returns some kind of default each time, which you then test. Can you see the problem?